third_party/incremental_dom/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright 2016 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Upstream Incremental DOM is written using ES6 modules. A gulp file is also
+# included which uses Rollup to generate a goog.module() style closure-library
+# compatible file:
+#
+#   https://github.com/google/incremental-dom/blob/master/gulpfile.js
+#
+# To avoid the runtime dependencies on gulp/node/rollup/etc this script
```

Comment:
Wow this rollup they're doing is annoying. I'm going to look into seeing if there's a better way to do this once this is submitted.